### PR TITLE
Share modal translations

### PR DESCRIPTION
**Pull request by curtisadams28**
[#7446](https://dev.azure.com/S72/SHIFT72/_workitems/edit/7446)

### CODE REVIEW

File: site/hr_HR.all.json [hunk 1]
```diff
@@ -135,16 +135,16 @@
     "other": "Udio"
   },
   "social_media_buttons_facebook": {
-    "other": "Podijeli na Facebooku"
+    "other": "Facebook"
   },
   "social_media_buttons_twitter": {
-    "other": "Podijelite na Twitteru"
+    "other": "Cvrkut"
```
Comment (PeculiarGoat) [inline site/hr_HR.all.json:141]: I can see Twitter being translated in a few of these files... that's not supposed to happen right?

File: site/ja_JP.all.json [hunk 2]
```diff
@@ -117,16 +117,16 @@
...
-    "other": "Twitterで共有する"
+    "other": "ツイッター"
   },
   "social_media_buttons_linkedin": {
-    "other": "Linkedinで共有する"
+    "other": "Linkedin"
   },
   "social_media_buttons_email": {
-    "other": "メールで共有する"
+    "other": "Eメール"
```
Comment (PeculiarGoat) [inline site/ja_JP.all.json:129]: Drop the E!

File: site/hr_HR.all.json (hunk 1 above)
Comment (curtisadams28) [reply site/hr_HR.all.json:141]: I looked at some old commits and it was translated in some files and not in others. I'll go through and change them to English.

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Few things to look at!

**Review by brynyShift72 — COMMENTED** [review]
for FR and EN remember to update in database and core-languages repo
https://dev.azure.com/S72/SHIFT72/_wiki/wikis/SHIFT72.wiki/214/Self-Service-Translation-update-process

**Review by PeculiarGoat — APPROVED** [review]
👍

**Comment by PeculiarGoat** [comment]
Don't forget the changelog!

**Review by smitsanghvi21 — COMMENTED** [review]
looks good to go after the merge conflict is resolved

**Review by brynyShift72 — APPROVED** [review]

**Pull request merged**